blocks/columns/columns.css

```diff
@@ -547,5 +547,35 @@
         font-size: var(--h2-base);
         line-height: 44px;
   }
+
+  .columns.columns-1-cols.banner-text {
```

Comment:
We have decided to remove nesting due to compatibility issues in Safari. Could you please remove nesting and push changes again?

Reply:
Okay @thomsebastin. Will do.